docs/handbook/01_intro.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+Quantum computing has the potential to revolutionize many fields in the 21st century. Over the past decade, numerous quantum computers from multiple providers based on different qubit technologies have been made publicly available.
+However, the best hardware is only as good as the software available to realize corresponding applications on it---a lesson learned from the past decades of research on designing and developing classical circuits and systems.
+Thanks to the software tools and methods for _Electronic Design Automation (EDA)_, we can create classical systems with a staggering amount of transistors and complex functionalities that we often take for granted.
+These methods allow designers to efficiently and automatically handle the intricacies of such systems and optimize their performance.
+Compared to that, most existing software solutions for quantum computing are based on manual approaches.
+This is not only susceptible to errors, inefficiency, and inconsistency but also leaves decades of research on design automation methods underutilized.
+
+The _Munich Quantum Toolkit (MQT)_, which is developed by the [Chair for Design Automation](https://www.cda.cit.tum.de/) at the [Technical University of Munich](https://www.tum.de/), aims to leverage this latent potential by providing a collection of state-of-the-art design automation methods and software tools for quantum computing.
+Our overarching objective is to provide solutions for design tasks across the entire quantum software stack.
+This entails high-level support for end users in realizing their _applications_ as well as efficient methods for the _classical simulation_, _compilation_, and _verification_ of quantum circuits.
+Reaching towards the hardware level, we also consider tools for _quantum error correction_ and _physical design_.
+In all these tools, we try to utilize _data structures and core methods_ facilitating the efficient handling of quantum computations.
+The proposed solutions demonstrate significant improvements in efficiency, scalability, and reliability.
```

Comment:
I know we don't have the space, nor is this the paper for it. But I don't know if one can claim this. Maybe soften the statement a little bit like: "The proposed solutions demonstrate how utilizing design automation expertise can have a positive impact on efficiency, scalability and reliability."

Reply:
Done in https://github.com/cda-tum/mqt/pull/10/commits/0f8a7402fe827d22aadaf7da4975c13205fe4a4c 👍🏼 
Adapted your version slightly. Please check if that's fine for you.